common/web/keyboard-processor/src/text/outputTarget.ts

```diff
@@ -446,6 +446,18 @@ export class Mock extends OutputTarget {
     this.text = this.getTextBeforeCaret() + s;
   }
 
+  /**
+   * Indicates if this Mock represents an identical context to that of another Mock.
+   * @param other
+   * @returns
+   */
+  isEqual(other: Mock) {
+    return this.text == other.text
+      && this.selStart == other.selStart
+      && this.selEnd == other.selEnd
+      && this.deadkeys().equal(other.deadkeys());
```

Comment:
If someone goes and makes a multitap key that only does deadkey stuff, the final check is required to be able to rewind deadkey manipulations.  Otherwise, we'd get a lot of excess deadkeys during such a multitap due to not erasing deadkeys from taps before the final one.